contracts/optimistic-ethereum/OVM/bridge/messaging/OVM_L1CrossDomainMessenger.sol

```diff
@@ -49,6 +49,7 @@ contract OVM_L1CrossDomainMessenger is iOVM_L1CrossDomainMessenger, Abs_BaseCros
     {
         require(address(libAddressManager) == address(0), "L1CrossDomainMessenger already intialized.");
         libAddressManager = Lib_AddressManager(_libAddressManager);
+        xDomainMessageSender = DEFAULT_XDOMAIN_SENDER;
```

Comment:
I think this should be a no-op given it happens in the constructor, but I think it'll only cost 200 gas here.

Reply:
This doesn't work with the L1 XDomain Messenger. I tried it, not sure why your constructor is not called.

Reply:
The initialize function is needed on at least one of them, because when you deploy the first one, you can't yet know the address of the other one.